packages/server-wallet/src/__test__/sync-objectives.test.ts

```diff
@@ -68,6 +68,82 @@ test('Objectives can be synced if a message is lost', async () => {
   });
 });
 
+test('handles the objective being synced even if no message is lost', async () => {
+  const createChannelParams: CreateChannelParams = createChannelArgs();
+
+  // We mimic not receiving a message containing objectives
+  const messageResponse = await a.createChannel(createChannelParams);
+
+  const channelId = messageResponse.channelResults[0].channelId;
+  const objectiveId = `OpenChannel-${channelId}`;
+  await b.pushMessage(getPayloadFor(bob().participantId, messageResponse.outbox));
+
+  // We expect both objectives to be there
+  expect(await getObjective(a.knex, objectiveId)).toBeDefined();
+  expect(await getObjective(b.knex, objectiveId)).toBeDefined();
+
+  const {outbox, newObjectives, channelResults} = await a.syncObjectives([objectiveId]);
+
+  // The only result will be a message containing the latest states
+  expect(outbox).toHaveLength(1);
+  expect(outbox[0]).toMatchObject({
+    method: 'MessageQueued',
+    params: {
+      recipient: 'bob',
+      sender: 'alice',
+      data: {
+        signedStates: [expect.objectContaining({turnNum: 0})],
+        requests: [],
+      },
+    },
+  });
+  expect(newObjectives).toHaveLength(0);
+  expect(channelResults).toHaveLength(0);
```

Comment:
These expectations should have the same behaviour no matter if you `b.pushMessage` or not, right?. The interesting thing to test is _what happens when `b` receives the objective again_, right?

Reply:
Oops, you're correct. Will correct.

Reply:
Resolved in e6a1605